server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -78,7 +78,13 @@ public enum MergeReason {
 
     public static final String SINGLE_MAPPING_NAME = Constants.DEFAULT_MAPPING_TYPE;
     public static final Setting<Long> INDEX_MAPPING_TOTAL_FIELDS_LIMIT_SETTING =
-        Setting.longSetting("index.mapping.total_fields.limit", 1000L, 0, Property.Dynamic, Property.IndexScope);
+        Setting.longSetting("index.mapping.total_fields.limit",
```

Comment:
Maybe this setting should be replicated and not changeable as it affects successful writes (replicated writes)?
(Changing it to a different value than the published table may result in unsuccessful, maybe unexpected replicated writes on the subscriber)